flatten_json.py

```diff
@@ -46,7 +46,9 @@ def _flatten(object_, key):
         :param key: carries the concatenated key for the object_
         :return: None
         """
-        if isinstance(object_, dict):
+        if not object_:
```

Comment:
This is great. Can you add a comment here describing why this is necessary for future reference please?